src/seed/factories/users.js

```diff
@@ -0,0 +1,30 @@
+import faker from 'faker'
+
+export default function (params) {
+  const {
```

Comment:
@roschaefer use this one to get propper defaults!

```js
  params = {
    name: faker.name.findName(),
    email: faker.internet.email(),
    password: '1234',
    avatar: faker.internet.avatar()
     ...params
  }
```

Reply:
I double checked empty values and given values and they work OK. I cannot reproduce that behaviour. Can you explain it maybe? Probably I did something wrong?

Reply:
@roschaefer https://runkit.com/appinteractive/nitro-backend-pull-88

Reply:
But it seems to work now.